api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -104,8 +188,12 @@ public MaterializeCohortResponse materializeCohort(@Nullable CohortReview cohort
       }
       criteria = new ParticipantCriteria(participantIds);
     }
-    QueryResult result = bigQueryService.executeQuery(bigQueryService.filterBigQueryConfig(
-        participantCounter.buildParticipantIdQuery(criteria, limit, offset)));
+    QueryJobConfiguration jobConfiguration;
+    TableQueryAndConfig tableQueryAndConfig = getTableQueryAndConfig(fieldSet);
+
+    jobConfiguration = fieldSetQueryBuilder.buildQuery(criteria, tableQueryAndConfig, limit, offset);
+    QueryResult result = bigQueryService.executeQuery(
```

Comment:
We should probably log the query if this fails, if the lower layer doesn't already do this.